tests/ses/nodes_preparation.pm

```diff
@@ -78,20 +78,20 @@ EOF
     systemctl 'stop apparmor';
```

Comment:
please don't touch this file I'm not sure who belongs...

Reply:
Fixed.